config/install/migrate_plus.migration.joomla_node.yml

```diff
@@ -42,4 +53,4 @@ migration_dependencies:
 dependencies:
   enforced:
     module:
-      - joomla_migrate
+      - joomla_migrate
```

Comment:
Every file should end with a new line. You deleted the blank line at the end of the file here.